### PR TITLE
add new video type for bilibili.com

### PR DESCRIPTION
**Pull request by WingGao**
fix the new type of `高清 1080P+` on bilibili.com
test command:
```bash
# you need a vip account
you-get --debug -c bilibili.txt --playlist -i "https://www.bilibili.com/video/av40115478"
```

### CODE REVIEW

File: src/you_get/extractors/bilibili.py [hunk 1]
```diff
@@ -111,6 +112,13 @@ def bilibili_space_video_api(mid, pn=1, ps=100):
     def bilibili_vc_api(video_id):
         return 'https://api.vc.bilibili.com/clip/v1/video/detail?video_id=%s' % video_id
 
+    @staticmethod
+    def url_size(url, faker=False, headers={},err_value=0):
+        try:
```
Comment (soimort) [inline src/you_get/extractors/bilibili.py]: Why is this wrapper function necessary? Under what circumstances can `url_size` from common.py fail?

Comment (WingGao) [reply src/you_get/extractors/bilibili.py]: Because sometime the server will return a `403` error for the audio file, but the video file is accessible.

**Comment by Tobyzgj** [comment]
Youku 201 does not have the right to play how to solve the problem？

**Comment by soimort** [comment]
:+1:

**Pull request merged**